CONTRIB.md

```diff
@@ -33,8 +33,15 @@ Apache NiFi MiNiFi C++ is a review then commit community. As a result, we will r
 review. We ask that you provide tests and documentation when possible. 
 
 Once you have completed your changes, including source code and tests, you can verify that you follow the Google style guide by running the following command:
-     $ make linter.
-     
+##### Linux or macOS
+```
+$ make linter
+```
+##### Windows
+```
+$ msbuild linter.vcxproj
```

Comment:
nitpicking, but this prompt should be `> `

Reply:
changed it in [e2a4e48](https://github.com/martinzink/nifi-minifi-cpp/commit/e2a4e487ce85305894d3ba7fc668f5d4f73a9821)